modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/RightSidebar/RightSidebarObjectFieldDetails.tsx

```diff
@@ -55,6 +59,26 @@ export function RightSidebarObjectFieldDetails() {
 
 	const {edges, nodes} = store.getState();
 
+	const objectDefinitionNodeData = selectedObjectDefinitionNode?.data as ObjectDefinitionNodeData;
+
+	const objectDefinition: Pick<
+		ObjectDefinition,
+		| 'accountEntryRestricted'
+		| 'accountEntryRestrictedObjectFieldName'
+		| 'externalReferenceCode'
+		| 'modifiable'
+		| 'name'
+		| 'status'
+	> = {
+		accountEntryRestricted: objectDefinitionNodeData.accountEntryRestricted,
+		accountEntryRestrictedObjectFieldName:
+			objectDefinitionNodeData.accountEntryRestrictedObjectFieldName,
+		externalReferenceCode: objectDefinitionNodeData.externalReferenceCode,
+		modifiable: objectDefinitionNodeData.modifiable,
+		name: objectDefinitionNodeData.name,
+		status: objectDefinitionNodeData.status,
+	};
```

Comment:
Hey @igor-franca, as we discussed, this will not be necessary as we will move all the logic to get the objectDefiniton inside the `EditObjectFieldContent.tsx`